### DISCUSSION TITLE
Helping with translations

### DISCUSSION THREAD

**Discussion by Kaemmelot** [firefly-iii/firefly-iii#5715] [General]
So I switched Firefly to German and noticed there doesn't seem to be any translated help pages (the question mark in the top right corner). This led to a few questions.
I noticed, that the link in the message `This help text is not yet available in your language. <a href="https://crowdin.com/project/firefly-iii-help">Will you help translate?</a>` points to an old crowdin project. Maybe you could change that?
Also the website [https://docs.firefly-iii.org/firefly-iii/support/contribute/#translations](https://docs.firefly-iii.org/firefly-iii/support/contribute/#translations) talks about a wiki page which leads to a 404. Does that still exist?
Then finally after I found the correct crowdin project on the website and created an account, I now fail to find the translations for the help pages. Are there any and if so, where can I find them? 😄

**Comment by JC5** [comment]
It so happens that in Firefly III 5.6.14 this has been changed and the link goes to the documentation at [docs.firefly-iii.org](https://docs.firefly-iii.org/), which is only available in English. 

I will make sure the documentation gets updated re: the 404!
